### ISSUE TITLE
Improve CLI

### ISSUE THREAD

**Issue by askibin** [askibin/perpetuals#51]
Add support for some instructions that are currently missing. Let CLI to load pool and custody configs from external files instead of being hard-coded,

**Comment by mzs-dev** [comment]
@theDefiBat

**Comment by theDefiBat** [comment]
yep can help on this, already using external config json files on our testing UI
